### PR TITLE
Update priority-accounts.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the navigation to add priority accounts.

fixes#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/10811

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

File: microsoft-365/admin/setup/priority-accounts.md [hunk 1]
```diff
@@ -53,19 +53,17 @@ The **Premium Mail Flow Monitoring** feature that's described in this topic is a
...
-Add priority accounts from the **Setup page**.
+Add priority accounts from the **Microsoft 365 Defender page**.
 
-1. Go to the Microsoft 365 admin center at <a href="https://go.microsoft.com/fwlink/p/?linkid=2024339" target="_blank">https://admin.microsoft.com</a>.
-
-2. Go to **Setup** > **Organizational knowledge**, and choose **View** under **Monitor your most important accounts**.
+1. Go to  Microsoft 365 Defender at <a href="https://security.microsoft.com/" target="_blank">https://security.microsoft.com/</a>.
 
-3. Select **Get Started** or **Manage**.
+2. Go to **Settings** > **Email and Collaboration** > **User Tags** > **Create**. T
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/setup/priority-accounts.md]:
```suggestion
2. Go to **Settings** > **Email and Collaboration** > **User Tags** > **Create**.
```

File: microsoft-365/admin/setup/priority-accounts.md [hunk 2]
```diff
@@ -53,19 +53,17 @@ The **Premium Mail Flow Monitoring** feature that's described in this topic is a
...
 
-4. On the **Add Priority accounts** page, in the search field, type the name or email address of the person you want to add to the priority accounts list. You can also set your email threshold for failed or delayed emails and get a weekly report of issues for priority accounts.
+3. The Priority Account tag is already created for you by default. By selecting **edit** you can add users to the priority tag. Then, **Add members** > **Add** to add members to the Priority account.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/setup/priority-accounts.md]:
```suggestion
3. The priority account tag is already created for you by default. By selecting **Edit** you can add users to the priority tag. Then, **Add members** > **Add** to add members to the priority account.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@kwekuako Gentle reminder.

**Pull request merged**